requirements.txt

```diff
@@ -27,7 +26,9 @@ PyJWT==0.1.2
 pyOpenSSL==16.1.0
 python-dateutil==2.1
 pytz==2014.10
+redis
```

Comment:
I'm planning on rewriting the requirements using Pipenv to manage explicit vs implicit pinning